docs/integrations-guides/custom_security.md

```diff
@@ -0,0 +1,73 @@
+- Threshold requirements (i.e, adversarial and liveness signing stake thresholds)
+- A [checkDACert](https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol#L8) function to verify the validity of an `EigenDACert` of the correct version
+- A [CertVersion](https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/contracts/src/periphery/cert/interfaces/IVersionedEigenDACertVerifier.sol#L12) to return the version (single digit "major" version) of the Cert, which is used by dispersing clients to know which version of the cert to build.
+
+The verification logic works uniformly for every quorum, meaning that custom quorums don't have any custom verification logic: they are verified exactly like the ETH and EIGEN quorums.
+Dispersals specify in the BlobHeader the [quorum_numbers](https://github.com/Layr-Labs/eigenda/blob/3e670ff3dbd3a0a3f63b51e40544f528ac923b78/api/proto/common/v2/common_v2.proto#L24) that are required to sign, and each quorum is then verified to have reached the required threshold.
+
+### EigenDACertVerifierRouter
+
+Given the immutable nature of the EigenDACertVerifier, changing its logic or thresholds requires deploying a new contract. Secure rollup stacks then need to securely point their derivation pipeline to the new contract, which would require a hard fork. To solve this problem, we introduce the EigenDACertVerifierRouter, which is a custom proxy contract which "reifies implementation update history" by maintaining a mapping of activation block numbers to CertVerifier contracts. This enabled onchain verification of historical certs, as well as offchain verification without requiring an archive node (since the entire mapping is stored in storage). It overall:
+- Maintains a mapping of activation block number (ABN) → `EigenDACertVerifier` address: new verifiers can be added at any future block
+- Routes `checkDACert` requests to the appropriate EigenDACertVerifier contract based on reference block number (RBN) in the abi.encoded cert
+
+Note that the Router is only compatible with EigenDACertVerifiers versions >= 3.
+
+Each rollup stack will need to integrate the Router in different ways to maintain a secure integration. We refer users to their respective rollup stack docs:
+- [OP Stack Integration](./rollup-guides/op-stack/README.md)
+- [Arbitrum Orbit Integration](./rollup-guides/orbit/overview.md)
+- [zkSync ZK Stack Integration](./rollup-guides/zksync/README.md)
+
+## Quick Start
+
+The flow to get setup with a CertVerifierRouter and enabling custom quorums would typically look like:
+1. Rollup integrates with EigenDA via the [proxy](./eigenda-proxy/eigenda-proxy.md), using the network-default [EigenDACertVerifier](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.8.1) deployed by EigenDA.
+2. Rollup decides it wants a custom quorum, or to change its security thresholds.
```

Comment:
shouldn't we also highlight or reference the process of bootstrapping a new custom quorum? After talking with @ian-shim this appears to be a slightly nontrivial amount of work

Reply:
Agree! But that's an eigenlayer level thing right? We prob have docs somewhere else for this? Or at least we should.

Reply:
Created https://linear.app/eigenlabs/issue/DAINT-558/docs-site-document-custom-quorum-bootstrapping